### PR TITLE
Ibillboard integration

### LINKED ISSUE #4628: Intent to implement support for iBILLBOARD AdServer
Hallo, our intent is implementing support for iBILLBOARD AdServer. We want to add support into amp-ad element.

CLA has been submitted. Corporation name: Internet BillBoard a.s.

### PR DESCRIPTION
**Pull request by pbilik**
iBillboard AMP integration is ready

Closes #4628.

### CODE REVIEW

**Comment by pbilik** [comment]
related to https://github.com/ampproject/amphtml/issues/4628

File: ads/ibillboard.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript, validateDataExists, validateSrcPrefix} from '../3p/3p';
+
+const validHosts = ['https://go.eu.bbelements.com', 'https://go.idnes.bbelements.com', 'https://go.goldbachpoland.bbelements.com', 'https://go.pol.bbelements.com', 'https://go.idmnet.bbelements.com'];
```
Comment (lannka) [inline ads/ibillboard.js]: nitpick: could you have one URL in one line?

**Comment by pbilik** [comment]
Hallo lannka,
I have fixed issues after your review.
I have not  implemented render-start API yet, because it is more complicated on our side and it requires modification of several parts of our adserver. Is it barrier for you to accept our integration? We will plan to implement render-start API later. Is it OK?
Can you help me with accepting our CLA in this pull request? We have corporate CLA. We created google users group refered in CLA. My email address (pavel.bilik@ibillboard.com) is added into this group. I have set my email in git configuration:
git config --global user.email
pavel.bilik@ibillboard.com
But there is still blocker: CLAs are signed, but unable to verify author consent ...
What can I do for removing this blocker?
Thank you very much.
Pavel

File: examples/ads.amp.html [hunk 2]
```diff
@@ -107,6 +107,7 @@
     <a href="#flite">Flite</a> |
     <a href="#genieessp"></a> |
     <a href="#gmossp" class="broken"></a> |
+    <a href="#ibillboard">iBILLBOARD AdServer</a> |
```
Comment (lannka) [inline examples/ads.amp.html]: just name it iBILLBOARD to save some spaces :-)

File: examples/ads.amp.html [hunk 3]
```diff
@@ -562,6 +563,12 @@ <h2 id="gmossp" class="broken">GMOSSP 320x50 banner</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="ibillboard">iBillboard 300x250 banner</h2>
+  <amp-ad width="300" height="250"
+          type="ibillboard"
```
Comment (lannka) [inline examples/ads.amp.html]: nit: please follow the same indentation as in this page.

File: ads/ibillboard.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+-->
+
+# iBILLBOARD AdServer
+
+Please visit [our website](https://www.ibillboard.com/) for more information.
+
+## Examples
+
+### Simple ad tag with `data-bn`
+
```
Comment (lannka) [inline ads/ibillboard.md]: Seems missing an example here?

File: examples/ads.amp.html (hunk 3 above)
Comment (pbilik) [inline examples/ads.amp.html]: done

File: ads/ibillboard.md (hunk 4 above)
Comment (pbilik) [inline ads/ibillboard.md]: removed

**Comment by pbilik** [comment]
CLA - corp. name is: Internet BillBoard a.s.
users group is: internet-billboard-git-contributors@googlegroups.com

**Comment by pbilik** [comment]
CLA: There are 2 commiters in our fork (pabi - pavel.bilik@ibillboard.com, jape - jan.petkov@billboard.cz).  Both emails have been added in the users group. Second email was added this morning. Yesterday I synchronized our fork with origin.

**Comment by lannka** [comment]
@cramforce could you help to force merge? Corp CLA looks good.

**Pull request merged**